### ISSUE TITLE
Examples depend on matplotlib but this isn't mentioned as requirement

### ISSUE THREAD

**Issue by gplssm** [oemof/demandlib#4]
I suggest to keep it easy and add matplotlib to the requirements. @oemof/demandlib what do you think?

**Comment by gplssm** [comment]
#5

**Comment by uvchik** [comment]
We lately had some problems installing matplotlib via pip. More requirements do not make it necessarily easier for us to maintain the package.

I would agree adding it by now but removing it immediately if problems occur again. 

I agree that I makes it easier for new users running the example out of the box.

**Comment by gplssm** [comment]
I agree with your thoughts!

Is the matplotlib problem you mentioned related with having installed matplotlib twice? Via
```
sudo apt-get install python3-matplotlib
```

and 
```
pip3 install matplotlib
```
?

At least in a virtualenv this is not a problem (for me).

**Comment by uvchik** [comment]
No, in the past `pip install matplotlib` just failed. But let's consider that it is a quite popular package and such problems will not occur often. So for me its fine until we know better.

**Comment by gplssm** [comment]
Closed by 546ef54fe3b1a43de3a478eba698e6788bd050d9
